app/mailers/invite_people_mailer.rb

```diff
@@ -11,14 +11,15 @@ def to_start_group(invitation:,
                      subject_params: {group_name: @invitation.invitable_name}
   end
 
-  def to_join_group(invitation: , locale: )
+  def to_join_group(invitation:, locale:, subject:)
```

Comment:
I'd default this to

```
def to_join_group(invitation:, locale:, subject: "email.to_join_group.subject")
   ...
end
```

so that we don't have to change the existing calls at all, but can override where we need to.